test/syscalls/linux/dev.cc

```diff
@@ -61,6 +61,16 @@ TEST(DevTest, LseekDevFull) {
   EXPECT_THAT(lseek(fd.get(), 123, SEEK_END), SyscallSucceedsWithValue(0));
 }
 
+TEST(DevTest, LseekDevKmsg) {
```

Comment:
Skip tests that open `/dev/ksmg/` on gVisor until we have the registration code on all paths. I'll explain what this means offline.

```
SKIP_IF(IsRunningOnGvisor());
```

Reply:
Actually, since this is part of `memdev`s, it'll be registered along with the rest of the `memdev` devices. Please disregard.